django_tables2/columns/templatecolumn.py

```diff
@@ -26,24 +27,27 @@ class TemplateColumn(Column):
     - *value*       -- value from `record` that corresponds to the current column
     - *default*     -- appropriate default value to use as fallback
     - *row_counter* -- The number of the row this cell is being rendered in.
+    - context variables pass as argument to column
```

Comment:
this one too. Also: please use the same style as the other list items.

Reply:
I changed the style because the keys are listed before, and here the value of the context key is dynamic. No static keys of context, so no bold.

Reply:
ah, makes sense! I would propose:
```
 - any context variables passed using the `extra_context` argument to `TemplateColumn`.
```